lightningd/opening_control.c

```diff
@@ -725,6 +726,146 @@ static void channel_config(struct lightningd *ld,
 	 ours->channel_reserve = AMOUNT_SAT(UINT64_MAX);
 }
 
+struct openchannel_hook_payload {
+	struct subd *openingd;
+	struct amount_sat funding_satoshis;
+	struct amount_msat push_msat;
+	struct amount_sat dust_limit_satoshis;
+	struct amount_msat max_htlc_value_in_flight_msat;
+	struct amount_sat channel_reserve_satoshis;
+	struct amount_msat htlc_minimum_msat;
+	u32 feerate_per_kw;
```

Comment:
Do we already know remote routing fees (base, ppm) here? If yes, we should add it to payload so plugin knows...

Reply:
No, we don't ever get told those directly; we learn them like everyone else, when/if it advertizes the channel.